notebooks/todos.md

```diff
@@ -0,0 +1,80 @@
+# Tests to run
```

Comment:
How up-to-date is this?  If it's not the current best place to look for plans, consider removing it from the pull request.